src/server/scripts/Northrend/IcecrownCitadel/boss_blood_queen_lana_thel.cpp

```diff
@@ -215,7 +215,6 @@ class boss_blood_queen_lana_thel : public CreatureScript
                     if (Creature* minchar = me->FindNearestCreature(NPC_INFILTRATOR_MINCHAR_BQ, 200.0f))
                     {
                         minchar->SetUInt32Value(UNIT_NPC_EMOTESTATE, 0);
-                        minchar->RemoveByteFlag(UNIT_FIELD_BYTES_1, UNIT_BYTES_1_OFFSET_ANIM_TIER, UNIT_BYTE1_FLAG_ALWAYS_STAND);
```

Comment:
Replace with  `minchar->SetAnimationTier(AnimationTier::Ground);`

Reply:
tested, ok

Reply:
If I recall correctly his spawn coordinates are in the air? If so, could be caused by https://github.com/TrinityCore/TrinityCore/pull/24875/files#diff-f3f953a63e2a008d22de69cb68f735c4R1992

Reply:
Why did you mark this as resolved?
> he is not on correct position while she is alive (he is on ground of first floor not over her head).

Reply:
added in todo